stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -830,6 +830,35 @@ extension Unsafe${Mutable}RawBufferPointer {
     let n = c / MemoryLayout<T>.stride
     return .init(start: .init(s._rawValue), count: n)
   }
+
+%  if Mutable:
+  @inlinable
+  @_alwaysEmitIntoClient
```

Comment:
FYI, @_alwaysEmitIntoClient implies @inlinable, so you only need to specify one or the other.

Reply:
The idea behind putting both is that @_aEIC doesn't have an obvious meaning, while @inlinable does.

Reply:
Ah, doing it for stylistic reasons is totally fine. I just want to make sure y'all had the right mental model, since @_aeic is not documented and all...